lib/rubocop/formatter/simple_text_formatter.rb

```diff
@@ -97,6 +87,48 @@ def message(offense)
         message = offense.corrected? ? green('[Corrected] ') : ''
         message << annotate_message(offense.message)
       end
+
+      # A helper class for building the report summary text.
+      class ReportSummarizer
+        include Colorizable, TextUtil
+
+        def initialize(file_count, offense_count, correction_count, rainbow)
+          @file_count = file_count
+          @offense_count = offense_count
+          @correction_count = correction_count
+          @rainbow = rainbow
+        end
+
+        def to_s
+          if correction_count > 0
+            "#{files} inspected, #{offenses} detected, #{corrections} corrected"
+          else
+            "#{files} inspected, #{offenses} detected"
+          end
+        end
+
+        private
+
+        attr_reader :file_count, :offense_count, :correction_count, :rainbow
```

Comment:
Any reason to keep those private? Seems to me we could have just used the fields directly.